shared/pages/CurrencyWallet/CurrencyWallet.js

```diff
@@ -61,8 +62,9 @@ export default class CurrencyWallet extends Component {
     }
   }
 
-  static getDerivedStateFromProps({ match: { params: { fullName } }, intl: { locale }, items, history }) {
+  static getDerivedStateFromProps({ match: { params: { fullName } }, intl: { locale }, items, history, tokens }) {
     const item = items.map(item => item.fullName.toLowerCase())
+    const token = tokens.map(item => item.fullName).includes(fullName.toUpperCase())
```

Comment:
```suggestion
    const token = tokens.map(item => item.fullName).includes(fullName.toUpperCase())
```

если это булевая переменная, то ее нужно называть isToken
а вообще, у нас же есть функция isETHToken, лучше ведь ее использовать

Reply:
Про функцию: если есть функция для этого, не нужно два раза одинаковый код писать. Более того, ты ради этого тащишь весь массив токенов в state, а это плохо для производительности и для понимания кода